### PR TITLE
Apparently multiple

### PR DESCRIPTION
**Pull request by satanasov**
Fixing new SGAPI use

Adding the new endpoint as per PR #9

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -213,4 +220,251 @@ app.get('/consensus/voted_power', function (req, res) {
...
+          res.send(output_str);
+        }));
+      }));
+    });
+});
+
 app.listen(PORT, () => console.log(`Listening on port ${PORT}!`));
+
+
+function addToOuptut(message)
```
Comment (sirkitree) [inline server.js:464]: What is this function for? It doesn't seem to be used and is spelled incorrectly I believe?

**Review by sirkitree — CHANGES_REQUESTED** [review]
Works as expected, just one question to answer before I merge. Thanks!

File: server.js (hunk 1 above)
Comment (satanasov) [reply server.js:464]: it's remnant from old variation of the code. I just forgot to delete it.

**Comment by sirkitree** [comment]
Thanks! Will send NCR today.

**Pull request merged**